src/Function.h

```diff
@@ -59,6 +59,30 @@ class CVQualifiers;
 
 ///////////////////////////////////////////////////////////////////////////////
 
+class FunctionAttribute
+{
+public:
+	string name;
+	int FuncAttrProb;
```

Comment:
Please follow a naming convention, unify these 2 declarations.

Reply:
Like, 
`string FuncAttrName`
`int FuncAttrProb` ?

Reply:
Dunno. Take a look what's the coding style of the `csmith` project.